src/client/browser/idle-page/status-markup-creator.js

```diff
@@ -0,0 +1,124 @@
+const SPINNER_CENTER_LEFT = 200;
+const SPINNER_CENTER_TOP  = 200;
+const SPINNER_RADIUS      = 196;
+
+const MESSAGE_BOX_TOP_INDENT  = 148;
+const MESSAGE_BOX_LEFT_INDENT = 17;
+
+const CONNECTED_SPINNER_COLOR    = '#28687F';
+const DISCONNECTED_SPINNER_COLOR = '#882E24';
+const SPINNER_GRADIENT_COLOR     = '#181818';
+
+const CONNECTED_TEXT    = 'CONNECTED';
+const DISCONNECTED_TEXT = 'DISCONNECTED';
+
+const PAGE_BACKGROUND_CLASS_NAME = 'page-background';
+const CONTAINER_CLASS_NAME       = 'container';
+const MESSAGE_BOX_CLASS_NAME     = 'message-box';
+const STATUS_ELEMENT_CLASS_NAME  = 'status';
+const CANVAS_CLASS_NAME          = 'spinner';
+
+const CONNECTED_CLASS_NAME    = 'connected';
+const DISCONNECTED_CLASS_NAME = 'disconnected';
+
+const ANIMATION_DELAY = 30;
+
+export default class StatusMarkupCreator {
+    constructor (connected) {
```

Comment:
Do we really need `connected` arg for the ctor? As I see, it's always used with `true` value on initializaton.